dockstore-webservice/src/main/resources/migrations.1.10.0.xml

```diff
@@ -85,4 +85,18 @@
             <column name="forumurl" type="varchar(255 BYTE)"/>
         </addColumn>
     </changeSet>
+    <changeSet author="nzhao (generated)" id="changeForumUrlType">
+        <modifyDataType columnName="forumurl" newDataType="varchar(256)" tableName="service"/>
+        <modifyDataType columnName="forumurl" newDataType="varchar(256)" tableName="workflow"/>
+    </changeSet>
+    <changeSet id="fixSevenBridgesImageLinks" author="gfjhogue">
```

Comment:
I think if you move your changes after this section, this won't show up as a diff.